app/services/package.js

```diff
@@ -112,7 +112,87 @@ module.exports.createFiscalDataPackage = function(attributes, resources) {
   });
 
   // JSON-LD
-  fdp['@context'] = 'http://schemas.frictionlessdata.io/fiscal-data-package.jsonld';
+  fdp['@context'] = 'http://schemas.frictionlessdata.io/' +
+    'fiscal-data-package.jsonld';
 
   return fdp;
 };
+
+function convertResource(resource, dataPackage, dataPackageUrl) {
+  var resourceUrl = resource.url || url.resolve(dataPackageUrl, resource.path);
+  return module.exports.createResourceFromSource(resourceUrl)
+    .then(function(result) {
+      // Copy some properties from original resource
+      // to keep them when re-assembling datapackage.json
+      result.name = resource.name;
+      result.dialect = resource.dialect;
+      result.encoding = resource.encoding;
+      result.source.url = resource.url;
+      result.source.fileName = resource.path;
```

Comment:
It doesn't matter. I just copy it to keep original value when re-assembling package. Path is processing properly (see line 122)